### PR TITLE
REST APY queries

### PR DESCRIPTION
**Pull request by cbrit**

### CODE REVIEW

File: x/cellarfees/client/rest/query.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/types/rest"
+	"github.com/peggyjv/sommelier/v4/x/cellarfees/types"
+)
+
+// QueryAccountRequestHandlerFn is the query APY REST Handler.
```
Comment (EricBolten) [inline x/cellarfees/client/rest/query.go:12]: Nit: comment leftover from copy-paste.

File: x/cellarfees/keeper/querier.go [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import (
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/peggyjv/sommelier/v4/x/cellarfees/types"
+)
+
+// NewQuerier returns a minting Querier handler.
```
Comment (EricBolten) [inline x/cellarfees/keeper/querier.go:12]: Nit: same comment issue here.

File: x/cellarfees/client/rest/query.go [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+func QueryAPYRequestHandlerFn(storeName string, clientCtx client.Context) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		clientCtx, ok := rest.ParseQueryHeightOrReturnBadRequest(w, clientCtx, r)
+		if !ok {
+			return
+		}
+
+		route := fmt.Sprintf("custom/%s/%s", types.QuerierRoute, types.QueryAPY)
```
Comment (EricBolten) [inline x/cellarfees/client/rest/query.go:20]: It looks like the route as reference elsewhere is `/cellarfees/apy`, is adding the `custom` prefix a requirement here? I assume that you were able to get the queries to return the correct data, but curious.

File: x/incentives/client/rest/query.go [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/types/rest"
+	"github.com/peggyjv/sommelier/v4/x/incentives/types"
+)
+
+// QueryAccountRequestHandlerFn is the query APY REST Handler.
```
Comment (EricBolten) [inline x/incentives/client/rest/query.go:12]: Nit: same comment thing here.

File: x/cellarfees/client/rest/rest.go [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+// REST query and parameter values
+const (
+	MethodGet = "GET"
+)
+
+// RegisterRoutes registers the cellarfees module REST routes.
+func RegisterRoutes(clientCtx client.Context, rtr *mux.Router, storeName string) {
+	r := rest.WithHTTPDeprecationHeaders(rtr)
+	r.HandleFunc(
+		"/cellarfees/apy", QueryAPYRequestHandlerFn(storeName, clientCtx),
```
Comment (EricBolten) [inline x/cellarfees/client/rest/rest.go:19]: Should we construct the argument to `HandleFunc` using the `types` values as we do elsewhere (`types.QuerierRoute` and `types.QueryAPY`)?

File: x/incentives/client/rest/rest.go [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+// REST query and parameter values
+const (
+	MethodGet = "GET"
+)
+
+// RegisterRoutes registers the incentives module REST routes.
+func RegisterRoutes(clientCtx client.Context, rtr *mux.Router, storeName string) {
+	r := rest.WithHTTPDeprecationHeaders(rtr)
+	r.HandleFunc(
+		"/incentives/apy", QueryAPYRequestHandlerFn(storeName, clientCtx),
```
Comment (EricBolten) [inline x/incentives/client/rest/rest.go:19]: Same question here as for `cellarfees` re: composing the argument.

**Review by EricBolten — COMMENTED** [review]
Some small questions and a bunch of nits.

File: x/cellarfees/client/rest/query.go (hunk 3 above)
Comment (cbrit) [reply x/cellarfees/client/rest/query.go:20]: I'm not totally sure of it's purpose, but it was included in the Cosmos SDK modules' rest implementations. My best guess is that it's an ABCI for application defined routes?

**Pull request closed without merging**